ts/Series/GeoHeatmap/GeoHeatmapSeries.ts

```diff
@@ -143,7 +204,45 @@ class GeoHeatmapSeries extends MapSeries {
              * @product   highmaps
              * @apioption plotOptions.geoheatmap.rowsize
              */
-            rowsize: 1
+            rowsize: 1,
+
+            stickyTracking: true,
+
+            /**
+             * Make the geoheatmap render its data points as an interpolated
+             * image. It can be used to show a Temperature Map-like charts.
+             *
+             * @sample maps/demo/geoheatmap-earth-statistics
+             *         Advanced demo of GeoHeatmap interpolation with multiple
+             *         datasets
+             *
+             * @type      {boolean}
+             * @default   false
+             * @since     @next
+             * @product   highmaps
+             * @apioption plotOptions.geoheatmap.interpolation
+             */
+            interpolation: false,
+
+            /**
+             * Represents how much blur should be added to the interpolated
+             * image. Works best in the range of 0-1, all higher values would
+             * need a lot more perfomance of the machine to calculate more
+             * detailed interpolation.
+             *
+             *  * **Note:** Useful, if the data is spread into wide range of
+             *  longitue and latitude values.
+             *
+             * @sample maps/series-geoheatmap/turkey-fire-areas
+             *         Simple demo of GeoHeatmap interpolation
+             *
+             * @type      {number}
+             * @default   1
+             * @since     @next
+             * @product   highmaps
+             * @apioption plotOptions.geoheatmap.interpolationBlur
+             */
+            interpolationBlur: 1
```

Comment:
The intension when we planned the `heatmap.interpoation` option, was that we first made it a simple boolean, but with the long-term possibility of expanding it as an object later. So instead of adding future options like `interpolationBlur`, `interpolationAlgorithm`, `interpolationSomething...`, we should now split it into two types.

A boolean enables or disables interpolation with default settings.

An object is used for enhanced control
```js
interpolation: {
    enabled: true,
    blur: 1
}
```